h2o-py/h2o/model/metrics_base.py

```diff
@@ -1470,8 +1470,8 @@ def _plot_pr(self, server=False, save_to_file=None, plot=True):
             if plt is None:
                 return decorate_plot_result(figure=RAISE_ON_FIGURE_ACCESS)
             fig = plt.figure()
-            plt.xlabel('Recall (TP/(TP+FP))')
-            plt.ylabel('Precision (TPR)')
+            plt.xlabel('Recall (TP/(TP+TN))')
```

Comment:
I know it is only 4 things, tp, fp, tn, fn but it is so easy to confused them!  Thank you for helping us out.  Just need to fix recall to tp/(tp+fn) and not fp.